agir/lib/utils.py

```diff
@@ -149,7 +149,7 @@ def validate_facebook_event_url(url):
     )
     # Regular expression for FB regular and short event URLs
     FACEBOOK_EVENT_URL_RE = re.compile(
-        r"^((?:https://)?(?:www\.)?(?:facebook|fb|m\.facebook)\.(?:com|me)/(?:events|e)/(?:\d\w{0,20}))(?:/.*)?$"
+        r"^((?:https://)?(?:www\.)?(?:facebook|fb|m\.facebook)\.(?:com|me)/(?:events|e)/(?:[\d\w]{0,20}))(?:/.*)?$"
```

Comment:
```suggestion
        r"^((?:https://)?(?:www\.)?(?:facebook|fb|m\.facebook)\.(?:com|me)/(?:events|e)/(?:\w{0,20}))(?:/.*)?$"
```
Du coup `\w` suffit non ?

Reply:
Non, les ids des événements sont des chiffres et les ids courts des lettres